test/e2e/tests/swaps/swap-erc20-with-loaded-state.spec.ts

```diff
@@ -0,0 +1,153 @@
+            'oneInch',
+            'pmm',
+            'zerion',
+            'lifi',
+            'socket',
+            'squid',
+            'sonarwatch',
+            'uniswapLabs',
+            'coinmarketcap',
+            'rango',
+          ],
+          occurrences: 13,
+          iconUrl:
+            'https://raw.githubusercontent.com/MetaMask/contract-metadata/master/images/weth.svg',
+          address: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+          name: 'Wrapped Ether',
+          decimals: 18,
+        },
+      })),
+    await mockServer
+      .forPost('https://transaction.api.cx.metamask.io/networks/1/getFees')
+      .thenCallback(() => ({
+        statusCode: 200,
+        json: {
+          blockNumber: 22017409,
+          id: 'b6d9c5f3-4fee-4470-be1b-6c574fe61315',
+          txs: [
+            {
+              cancelFees: [],
+              return: '0x',
+              status: 1,
+              gasUsed: 188186,
+              gasLimit: 241302,
+              fees: [
+                {
+                  maxFeePerGas: 6393950816,
+                  maxPriorityFeePerGas: 1000000004,
+                  gas: 241302,
+                  balanceNeeded: 1542873120043734,
+                  currentBalance: 30009434625664560,
+                  error: '',
+                },
+              ],
+              feeEstimate: 821886724082654,
+              baseFeePerGas: 3367416938,
+              maxFeeEstimate: 1542873119802432,
+            },
+          ],
+        },
+      })),
+  ];
+}
+
+describe('Swap', function () {
+  it('should swap WETH to ETH', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withNetworkControllerOnMainnet()
+          .withTokensController({
+            allTokens: {
+              '0x1': {
+                '0x5cfe73b6021e818b776b421b1c4db2474086a7e1': [
+                  {
+                    address: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+                    symbol: 'WETH',
+                    decimals: 18,
+                    isERC721: false,
+                    aggregators: [],
+                  },
+                ],
+              },
+            },
+          })
+          .build(),
+        title: this.test?.fullTitle(),
+        testSpecificMock: mockSwapQuotes,
+        localNodeOptions: [
+          {
+            type: 'anvil',
+            options: {
+              chainId: 1,
+              hardfork: 'london',
+              loadState:
+                './test/e2e/seeder/network-states/swap-state/withSwapContracts.json',
+            },
+          },
+        ],
+      },
+      async ({ driver, localNodes }) => {
+        await loginWithBalanceValidation(driver, localNodes[0]);
+
+        const homePage = new HomePage(driver);
+        await homePage.check_pageIsLoaded();
+        await homePage.check_expectedTokenBalanceIsDisplayed('50', 'WETH');
+
+        // disable smart transactions
```

Comment:
:information_source:  we cannot use fixtures in this case, as there's a migration that overwrites the STX value to set it ON by default, so we need to go to settings to disable the toggle